src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataReadMeControl.xaml.cs

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using Microsoft.VisualStudio.Markdown.Platform;
+using NuGet.PackageManagement.UI.ViewModels;
+using NuGet.VisualStudio;
+using NuGet.VisualStudio.Telemetry;
+
+namespace NuGet.PackageManagement.UI
+{
+
+    /// <summary>
+    /// Interaction logic for PackageMetadataReadMeControl.xaml
+    /// </summary>
+    public partial class PackageMetadataReadMeControl : UserControl, IDisposable
+    {
+        public static readonly DependencyProperty PackageMetadataProperty =
+            DependencyProperty.Register(
+                nameof(PackageMetadata),
+                typeof(DetailedPackageMetadata),
+                typeof(PackageMetadataReadMeControl),
+                new PropertyMetadata(OnPropertyChanged));
+
+#pragma warning disable CS0618 // Type or member is obsolete
+        private IMarkdownPreview _markdownPreview;
+#pragma warning restore CS0618 // Type or member is obsolete
+        private bool _disposed = false;
+
+        private ReadMePreviewViewModel ReadMeViewModel { get => (ReadMePreviewViewModel)DataContext; }
+
+        [Obsolete]
```

Comment:
I am curious, why add an obsolete member to the new code? It could be possible that I am not aware of the MVVM coding guidelines.

Reply:
Good catch, I think I was running something in there before and it's no longer necessary.